tests/pkg_fatfs/main.c

```diff
@@ -126,15 +141,13 @@ static int _mk(int argc, char **argv)
             puts("[OK]\n");
             return 0;
         }
-        else {
-            printf("[FAILED] (f_close error %d)\n", close_resu);
-            return -2;
-        }
-    }
-    else {
-        printf("failed to create file '%s'\n", argv[1]);
+
+        printf("[FAILED] (f_close error %d)\n", close_resu);
+        return -2;
     }
-    return 0;
+
+    printf("failed to create file '%s'\n", argv[1]);
```

Comment:
minor nit here: all other error messages start with `[FAILED] `, I suggest to adapt wording accordingly?